src/sql/Sql.cpp

```diff
@@ -5,4 +5,43 @@ namespace bakcyl::sql {
     Sql::Sql() {
 	
     };
+    
+    std::vector<Product> Sql::getAllProducts() const {
+        std::vector<Product> products;
+
+        {
```

Comment:
wstawiane tego w nowym bloku ma jakiś konkretny powód?

Reply:
W dokumentacji było napisane, że usuwanie bazy danych może nastąpić tylko, kiedy reszta obiektów (kwerendy itp.) będą poza Scopem i tam właśnie tak to pisali.

https://doc.qt.io/qt-5/qsqldatabase.html#removeDatabase